packages/circuits/scripts/buildParallel.mjs

```diff
@@ -3,8 +3,10 @@ import os from 'os'
 import url from 'url'
 import path from 'path'
 import { circuitContents } from './circuits.mjs'
+import fs from 'fs'
 
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
+const buildDir = path.join(__dirname, '../zksnarkBuild.tmp')
```

Comment:
Should use a proper [temp dir](https://nodejs.org/api/fs.html#fspromisesmkdtempprefix-options).

Reply:
This is moot because we won't use this directory, but anytime you add a directory like this you should add it to `.gitignore` so we don't accidentally commit it.

Reply:
using `os.tmpdir()` now